boards.txt

```diff
@@ -1854,6 +1854,223 @@ tinypico.menu.EraseFlash.none.upload.erase_cmd=
 tinypico.menu.EraseFlash.all=Enabled
 tinypico.menu.EraseFlash.all.upload.erase_cmd=-e
 
+##############################################################
+
+aventen_s3_sync.name=Aventen S3 Sync
+aventen_s3_sync.vid.0=0x303a
+aventen_s3_sync.pid.0=0x1001
+## Based upon ESP32-S3 Dev Board
+
+aventen_s3_sync.bootloader.tool=esptool_py
+aventen_s3_sync.bootloader.tool.default=esptool_py
+
+aventen_s3_sync.upload.tool=esptool_py
+aventen_s3_sync.upload.tool.default=esptool_py
+aventen_s3_sync.upload.tool.network=esp_ota
+
+aventen_s3_sync.upload.maximum_size=1310720
+aventen_s3_sync.upload.maximum_data_size=327680
+aventen_s3_sync.upload.flags=
+aventen_s3_sync.upload.extra_flags=
+aventen_s3_sync.upload.use_1200bps_touch=false
+aventen_s3_sync.upload.wait_for_upload_port=false
+
+aventen_s3_sync.serial.disableDTR=false
+aventen_s3_sync.serial.disableRTS=false
+
+aventen_s3_sync.build.tarch=xtensa
+aventen_s3_sync.build.bootloader_addr=0x0
+aventen_s3_sync.build.target=esp32s3
+aventen_s3_sync.build.mcu=esp32s3
+aventen_s3_sync.build.core=esp32
+aventen_s3_sync.build.variant=esp32s3
+aventen_s3_sync.build.board=ESP32S3_DEV
+
+aventen_s3_sync.build.usb_mode=1
+aventen_s3_sync.build.cdc_on_boot=0
+aventen_s3_sync.build.msc_on_boot=0
+aventen_s3_sync.build.dfu_on_boot=0
+aventen_s3_sync.build.f_cpu=240000000L
+aventen_s3_sync.build.flash_size=4MB
```

Comment:
Idk what flash size you have on board. But here you have default 4MB, but later in config you have only 16 MB option.
So from that I assume this should be changed to 16MB if I am correct.
```suggestion
aventen_s3_sync.build.flash_size=16MB
```